interchain-token-service/src/events.rs

```diff
@@ -0,0 +1,97 @@
+use cosmwasm_std::{Attribute, Event};
+use router_api::{Address, ChainName, CrossChainId};
+
+use crate::primitives::ItsMessage;
+
+pub enum ItsContractEvent {
+    ItsMessageReceived {
+        cc_id: CrossChainId,
+        destination_chain: ChainName,
+        message: ItsMessage,
+    },
+    ItsAddressSet {
+        chain: ChainName,
+        address: Address,
+    },
+    ItsAddressRemoved {
+        chain: ChainName,
+    },
```

Comment:
can we use different verbs than set and remove (everywhere, not just for the event). It's not just a store entry, it also implies access control, right?

Reply:
e.g. allow, register, authorize

Reply:
Yeah, I'll use register/deregister for consistency